docs/source/notebooks/Bayes_factor.ipynb

```diff
@@ -180,7 +180,7 @@
     "\n",
     "$$p(y) \\propto \\frac{B(\\alpha + h,\\ \\beta + n - h)} {B(\\alpha, \\beta)}$$\n",
     "\n",
-    "This expresion have been coded in the following cell,but with a twist. We will be using the `betaln` function instead of the `beta` function, this is done to prevent underflow."
+    "This expresion have been coded in the following cell, but with a twist. We will be using the `betaln` function instead of the `beta` function, this is done to prevent underflow."
```

Comment:
```suggestion
    "This expression have been coded in the following cell, but with a twist. We will be using the `betaln` function instead of the `beta` function, this is done to prevent underflow."
```

Reply:
Minor nit:  "This expression has [not "have"] been coded..."